### PR TITLE
Update workflow to use Go 1.18 and Go 1.17

### PR DESCRIPTION
**Pull request by gmlewis**

### CODE REVIEW

**Comment by gmlewis** [comment]
Awaiting LGTM+Approval from any contributor on this repo before merging.

**Review by raynigon — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @raynigon !
Merging.

**Pull request merged**
